### ISSUE TITLE
Improvement to typingtools: Make the viral taxa keywords more clear, better explain which species are actually typed and up to which level/which limitations.

### ISSUE THREAD

**Issue by DennisSchmitz** [DennisSchmitz/Jovian_archive#68]
Currently the keywords of the different virus typing-tools are based on their historical names, e.g. the Norovirus (NoV) typing-tool used to only type Noroviruses. However, now it actually types the GE-causing species within the Caliciviridae (Sapovirus en Norwalk virus). A more fitting and more correct name would be to name it `Caliciviridae` typing-tool. Similar issues for the Enterovirus (EV) typing-tool which actually types most/all GE-causing species within the Picornaviridae which incidentally also makes the separate Hepatitis A (HAV) typing-tool obsolete.

We also need better explanation of exactly which species are typed by the different typing-tools and their limitations. However, this is developed independently by Harry Vennema, and I've asked him for this information and will update it accordingly.

So basically, functionality doesn't need to change but only better explained.

**Comment by DennisSchmitz** [comment]
WIP, closing
